weblate/trans/util.py

```diff
@@ -395,3 +395,8 @@ def count_words(string: str, language: Language | None = None) -> int:
                 even = not even
         return count
     return sum(len(s.split()) for s in split_plural(string) if not is_unused_string(s))
+
+
```

Comment:
Is this the correct file for this function?

Reply:
Yes, it could be placed here.

Reply:
Looking again at it, placing it next to `format_html_join_comma` might be a better fit.